{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "40792127",
  "multipleFaultNumbers": null,
  "date": "2020-12-29T03:53:03",
  "region": "Bolton",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-12-29T03:53:03",
  "information": "",
  "actualTimeOfRestoration": "2020-12-29T03:53:03",
  "outageCentrePoint": {
    "lat": 53.563637,
    "lng": -2.378298
  },
  "outageLocations": null,
  "AffectedPostcodes": " BL3 1DJ, BL3 1DR, BL3 1DT",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}